### PR TITLE
Add entity activity feed without real diffs

### PR DESCRIPTION
**Pull request by matthew-white**

### CODE REVIEW

File: src/components/entity/activity.vue [hunk 1]
```diff
@@ -29,13 +30,55 @@ export default {
...
+  // There won't be an entity.create event if the entity was created before
+  // v2023.3. In that case, we create an artifical event to show in the feed.
+  if (!includesCreate) {
+    const entry = {
+      action: 'entity.create',
+      actor: entity.creator,
+      loggedAt: entity.createdAt
+    };
+    result.push({ entry });
+  }
```
Comment (matthew-white) [inline src/components/entity/activity.vue:75]: Remove.

File: src/components/entity/activity.vue [hunk 2]
```diff
@@ -29,13 +30,55 @@ export default {
...
 
 import { useRequestData } from '../../request-data';
 
 // The component does not assume that this data will exist when the component is
 // created.
-const { audits, diffs, resourceStates } = useRequestData();
-const { initiallyLoading, dataExists } = resourceStates([audits, diffs]);
+const { entity, audits, diffs, resourceStates } = useRequestData();
+const { initiallyLoading } = resourceStates([audits, diffs]);
+const { dataExists } = resourceStates([entity, audits, diffs]);
```
Comment (matthew-white) [inline src/components/entity/activity.vue:45]: Revert.

File: src/components/entity/feed-entry.vue [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+            </span>
+          </template>
+          <template #name><actor-link :actor="entry.actor"/></template>
+        </i18n-t>
+      </template>
+      <template v-if="entry.action === 'entity.create'">
+        <span class="icon-magic-wand"></span>
+        <!-- entry.actee == null for an artifical entity.create event for an
+        entity created before v2023.3. -->
+        <i18n-t v-if="entry.actee == null || entry.details.submissionCreate != null"
```
Comment (matthew-white) [inline src/components/entity/feed-entry.vue:50]: Remove case for artificial event.

**Comment by matthew-white** [comment]
To do: add tests of `EntityActivity` in a future PR, once #792 is merged.

File: test/components/entity/feed-entry.spec.js [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+      requestData: testRequestData([useEntity], {
+        entity: testData.extendedEntities.last()
+      })
+    }
+  }));
+
+describe('EntityFeedEntry', () => {
+  beforeEach(() => {
+    mockLogin({ displayName: 'Alice' });
+    testData.extendedEntities.createPast(1, { label: 'dogwood' });
```
Comment (matthew-white) [inline test/components/entity/feed-entry.spec.js:33]: `uuid: 'e'`

File: test/components/entity/feed-entry.spec.js [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+      });
+
+      it('links to the dataset', () => {
+        const title = mountComponent().get('.feed-entry-title');
+        const { to } = title.getComponent(RouterLinkStub).props();
+        to.should.equal('/projects/1/datasets/trees');
+      });
+    });
+
+    describe('artificial entity.create event', () => {
```
Comment (matthew-white) [inline test/components/entity/feed-entry.spec.js:185]: Delete.

File: test/components/entity/feed-entry.spec.js [hunk 6]
```diff
@@ -0,0 +1,304 @@
...
+    });
+    const margin = component.findAllComponents(EntityFeedEntry)
+      .map(({ element }) => {
+        const { marginTop, marginBottom } = getComputedStyle(element);
+        return {
+          top: marginTop === '' ? 0 : Number.parseFloat(marginTop),
+          bottom: marginBottom === '' ? 0 : Number.parseFloat(marginBottom)
+        };
+      });
+    margin[0].top.should.equal(0);
```
Comment (matthew-white) [inline test/components/entity/feed-entry.spec.js:299]: Assert that `margin.length === 3`.

**Review by ktuite — APPROVED** [review]

**Review by matthew-white — COMMENTED** [review]
Notes from interactive code review

File: src/components/entity/activity.vue [hunk 7]
```diff
@@ -29,13 +30,55 @@ export default {
...
+
+const feed = computed(() => {
+  const result = [];
+  let diffIndex = 0;
```
Comment (matthew-white) [inline src/components/entity/activity.vue:49]: This needs to be reversed.

**Pull request merged**